src/index.html

```diff
@@ -5,42 +5,53 @@
   <title>Activity Tracker</title>
   <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Open+Sans:400,800&display=swap"
     rel="stylesheet" />
-</head>
+    <link href="https://fonts.googleapis.com/css?family=Questrial&display=swap" rel="stylesheet">
+  </head>
 <body>
-  <h1 id="headerText">Activity Tracker</h1>
+  <div id= "header-border">
+    <h1 id="header-text">FITT<span id="tracker">RACKER</span><span id="period">.</span></h1>
+  </div>
   <section class="main">
     <section class="main-sidebar-body">
       <section class="body-sidebar-sidebarContainer">
-        <h2 class="sidebar-header-name" id="sidebarName">Name Name</h2>
-        <div class="sidebar-header-line"></div>
         <img src="./images/The Rock.jpg" class="sidebar-header-userImage"></img>
         <div class="sidebar-header-line"></div>
-        <p class="sidebar-header-userInfo" id="userAddress"></p>
-        <div class="sidebar-header-line"></div>
-        <p class="sidebar-header-userInfo" id="userEmail"></p>
-        <div class="sidebar-header-line"></div>
-        <p class="sidebar-header-userInfo" id="userStridelength"></p>
+        <div class="sidebar-header-userInfo">
+          <p class="sidebar-header-userInfo" id="sidebarName">Name Name</p>
         <div class="sidebar-header-line"></div>
-        <p class="sidebar-header-userInfo" id="stepGoalCard"></p>
-        <p class="sidebar-header-userInfo" id="avStepGoalCard">
-          <p>
+          <p class=sidebar-header-userInfo id="userAddress"></p>
+          <div class="sidebar-header-line"></div>
+          <p class="sidebar-header-userInfo" id="userEmail"></p>
+          <div class="sidebar-header-line"></div>
+          <p class="sidebar-header-userInfo" id="userStridelength"></p>
+          <div class="sidebar-header-line"></div>
+          <p class="sidebar-header-userInfo" id="stepGoalCard"></p>
+          <p class="sidebar-header-userInfo" id="avStepGoalCard"><p>
+        </div>
             <div class="sidebar-header-line"></div>
-            <section class="sidebar-body-friendContainer">
-              <p class="sidebar-header-userInfo">Friends</p>
-              <section class="sidebar-friendContainer-listItems">
+            <!-- <section class="sidebar-body-friendContainer">
```

Comment:
Generally don't push commented code to master

Reply:
gotcha! i'm going back into it